cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -488,6 +489,32 @@ class alignas(16) mutable_column_device_view
         detail::column_device_view_base::null_mask());
   }
 
+  /**---------------------------------------------------------------------------*
+   * @brief Return first element (accounting for offset) after underlying data
+   * is casted to the specified type.
+   *
+   * @tparam T The desired type
+   * @return T* Pointer to the first element after casting
+   *---------------------------------------------------------------------------**/
+  template <typename T, std::enable_if_t<is_fixed_width<T>()> * = nullptr>
+  T* begin() const {
```

Comment:
Should add an `end` too.

Reply:
Just an FYI, an alternative way you can use `enable_if_t` is like this:
```c++
template <typename T>
std::enable_if_t<is_fixed_width<T>(), T*> 
begin()
```
i.e., you can use it on the return type of the function. Which may be a bit more concise in this instance.